inference-engine/tests/functional/plugin/cpu/shared_tests_instances/subgraph_tests/range_add.cpp

```diff
@@ -0,0 +1,31 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vector>
+
+#include "subgraph_tests/range_add.hpp"
+#include "common_test_utils/test_constants.hpp"
+
+using namespace LayerTestsDefinitions;
+
+namespace {
+
+const std::vector<float> start = { 1.0f, 1.2f };
```

Comment:
maybe to extend with negative numbers in range?

Reply:
Done